src/OpenLoco/Interop/Hooks.cpp

```diff
@@ -498,8 +498,10 @@ static void registerMemoryHooks()
     // Hook Locomotion's alloc / free routines so that we don't
     // allocate a block in one module and freeing it in another.
     writeJmp(0x4d1401, (void*)&fn_malloc);
+    writeJmp(0x406bf7, (void*)&fn_malloc);
     writeJmp(0x4D1B28, (void*)&fn_realloc);
```

Comment:
Please add one more for realloc just in case

Reply:
Do you happen to know which function address wraps `realloc` in loco.exe? I just found the two for malloc and free

Reply:
I found 0x406C02 and have wrapped it as well. With this, the game still runs fine, so hooking it just in case.